README_ru.md

```
@@ -0,0 +1,415 @@
+- [.env](./docker/.env) — базовые настройки системы (`SVR_HTTP_PORT`, `MYSQL_PASSWORD`, `MINIO_PASSWORD` и др.).
+- [service_conf.yaml.template](./docker/service_conf.yaml.template) — конфигурация бэкенд-сервисов. Переменные окружения подставляются автоматически при старте контейнера.
+- [docker-compose.yml](./docker/docker-compose.yml) — оркестрация сервисов.
+
+> Подробное описание переменных и настроек есть в [./docker/README](./docker/README.md). Их можно использовать как `${ENV_VARS}` в `service_conf.yaml.template`.
+
+Чтобы изменить HTTP-порт по умолчанию (80), в [docker-compose.yml](./docker/docker-compose.yml) замените `80:80` на `<ВАШ_ПОРТ>:80`.
+
+После изменений перезапустите контейнеры:
+
+```bash
+$ docker compose -f docker-compose.yml up -d
```

markdownlint-cli2 (0.23.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)